app/views/pets/edit.html.erb

```diff
@@ -3,14 +3,14 @@
   <%= form_tag "/pets/#{@pet.id}", method: :patch do %>
     <%= label_tag "Image"%>
     <%= text_field_tag :image, @pet.image%><br/>
-    <%= label_tag "Name" %>
+    <%= label_tag "Name (required)" %>
     <%= text_field_tag :name, @pet.name %><br/>
-    <%= label_tag "Description" %>
-    <%= text_field_tag :description, @pet.description %><br/>
-    <%= label_tag "Age" %>
+    <%= label_tag "Age (required)" %>
     <%= text_field_tag :age, @pet.age %><br/>
-    <%= label_tag "Sex" %>
+    <%= label_tag "Sex (required)" %>
     <%= text_field_tag :sex, @pet.sex %><br/><br/>
+    <%= label_tag "Description" %>
+    <%= text_field_tag :description, @pet.description %><br/>
```

Comment:
tony's suggestion. description was splitting the required fields so i moved it below the others